spec/domain/people/neuanmeldungen/approve_spec.rb

```diff
@@ -50,6 +50,7 @@ def expect_role(role, expected_role_class, expected_group)
       .and change { neuanmeldung_approved_role_class.count }.by(3)
       .and change { ExternalInvoice::SacMembership.count }.by(3)
       .and change { Delayed::Job.where("handler like '%CreateInvoiceJob%'").count }.by(3)
+      .and have_enqueued_mail(People::NeuanmeldungenMailer, :approve).exactly(3).times
```

Comment:
Bitte in der spec noch prüfen, dass hier der Namen der sektion als gruppen namen verwendet wird

Reply:
auch für reject